packages/open-next/package.json

```diff
@@ -17,7 +17,8 @@
     "dev": "tsc -w"
   },
   "exports": {
-    "./*": "./dist/*"
+    "default": "./dist/*",
+    "types": "./dist/*.d.ts"
```

Comment:
Shouldn't this be
```suggestion
    "./*": {
      "default": "./dist/*",
      "types": "./dist/*.d.ts"
    }
```

Reply:
You are totally correct. Missed copying the full diff from my patch-package patch. 😅